lib/Doctrine/ORM/Tools/EntityGenerator.php

```diff
@@ -335,6 +336,11 @@ public function __construct(<params>)
 }
 ';
 
+    /**
+     * @var \Doctrine\Inflector\Inflector
+     */
+    protected $inflector;
```

Comment:
```suggestion
    private $inflector;
```

Reply:
No, in this case its used from protected methods, maybe someone overwrites them and needs access.

Reply:
Maybe we could make the protected methods final?